stages/org.osbuild.rhsm.facts

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+"""
+The `org.osbuild.rhsm.facts` stage is allows for storing rhsm-facts format
+artifacts in the built image. Keys and values are supplied in options.
+"""
+
+import json
+import os
+import sys
+
+import osbuild.api
+
+
+SCHEMA = r"""
+"definitions": {
+  "key": {
+    "type": "string"
+  },
+  "value": {
+    "type": "string"
+  }
+},
```

Comment:
You can just inline that, or is there a particular reason to have this as separate definitions?

Reply:
There's no specific reason I was planning on perhaps adding a pattern for the allowed RHSM key structure but that's premature.